pkg/operator/controller/sync-http-error-code-configmap/controller.go

```diff
@@ -125,7 +125,7 @@ type reconciler struct {
 // userConfigMapToIngressController maps a user-created error-page configmap to
 // a slice of reconcile requests, one request per ingresscontroller that
 // references the configmap for custom error pages.
-func (r *reconciler) userConfigMapToIngressController(o client.Object) []reconcile.Request {
+func (r *reconciler) userConfigMapToIngressController(ctx context.Context, o client.Object) []reconcile.Request {
 	requests := []reconcile.Request{}
 	controllers, err := r.ingressControllersWithConfigMap(o.GetName())
```

Comment:
`ingressControllersWithConfigMap` lists ingress controllers and uses `Background` context. We can now pass the real context to it.